### ISSUE TITLE
Error in file(file, "rt") : invalid 'description' argument

### ISSUE THREAD

**Issue by efegidri** [StevenVB12/patternize#33]
Hello! I am trying to run the "Heliconius erato erato verus hybrid" example using  the script from github but when I run the function:
"plotHeat(summedRaster_regRGBEra, IDListEra, plotCartoon = TRUE, refShape = 'target', outline = outline_BC0057,lines = lines_BC0057, flipRaster = 'xy',imageList = imageListEra, cartoonID = 'BC0057', cartoonFill = 'black', cartoonOrder = 'under',colpalette = colfunc, refImage = target)"
I get the next error "Error in file(file, "rt") : invalid 'description' argument"
Someone could help me with the solution? thank in advance, FEDE

**Comment by StevenVB12** [comment]
Hi Fede,

Sorry you’re experiencing problems. I’m unable to reproduce your error. Did you download all the necessary files from GitHub and when you load the data in the earlier lines of code, do you get any errors?

Steven

From: efegidri ***@***.***>
Sent: Thursday, September 8, 2022 7:38 PM
To: StevenVB12/patternize ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [StevenVB12/patternize] Error in file(file, "rt") : invalid 'description' argument (Issue #33)


Hello! I am trying to run the "Heliconius erato erato verus hybrid" example using the script from github but when I run the function:
"plotHeat(summedRaster_regRGBEra, IDListEra, plotCartoon = TRUE, refShape = 'target', outline = outline_BC0057,lines = lines_BC0057, flipRaster = 'xy',imageList = imageListEra, cartoonID = 'BC0057', cartoonFill = 'black', cartoonOrder = 'under',colpalette = colfunc, refImage = target)"
I get the next error "Error in file(file, "rt") : invalid 'description' argument"
Someone could help me with the solution? thank in advance, FEDE

**Comment by efegidri** [comment]
Hi steven,
yes, I downloaded all the required files from GitHub. When I run the function  "plotHeat"  I get the error "Error in file(file, "rt"): invalid 'description' argument " but, it could be something in the R program? because I recently began to get many different errors running a script that previously I could run...
thanks in advance, FEDE

**Comment by StevenVB12** [comment]
What R version are you using? I heard from someone else that new errors occur when using R 4.2.1. I already made an update in the code that solved it for me.

Steven

**Comment by efegidri** [comment]
yes, I am using version R 4.2.1. "I already made an update in the code that solved it for me.", ok, I will try to run the script again. thank you. Best,  FEDE

**Comment by StevenVB12** [comment]
Ok. Please let me know how it goes!

Thanks,

Steven
